samples/04_gis_analysts_data_scientists/finding_suitable_spots_for_AED_devices_using_raster_analytics.ipynb

```diff
@@ -44,20 +44,20 @@
   },
```

Comment:
<p>i think this piece of code is not required. Can you kindly check. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1021/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>